Documentation/English/Supported.md

```diff
@@ -0,0 +1,194 @@
+
+# Supported
```

Comment:
This is vital information and should stay in README.md.
There are two things I would consider moving to a sub-page: the contributor lists, and perhaps the Configuration section.

Reply:
OK~